.github/workflows/multi-arch-build.yaml

```diff
@@ -65,59 +66,51 @@ jobs:
           test -n "$VERSION"
           echo "::set-output name=version::${VERSION}"
 
-      # Generate image FQINs, labels, check whether to push
-      - name: Generate image information
+      - name: Generate image FQIN(s)
         id: image_info
         run: |
-          VERSION='v${{ steps.sniff_test.outputs.version }}'
           # workaround vim syntax-hilighting bug: '
           if [[ "${{ matrix.source }}" == 'stable' ]]; then
-            # quay.io/podman/stable:vX.X.X
+            VERSION='v${{ steps.sniff_test.outputs.version }}'
             ALLTAGS=$(skopeo list-tags \
                       docker://$PODMAN_QUAY_REGISTRY/stable | \
                       jq -r '.Tags[]')
-            PUSH="false"
-            if ! fgrep -qx "$VERSION" <<<"$ALLTAGS"; then
-                PUSH="true"
-            fi
 
-            FQIN="$PODMAN_QUAY_REGISTRY/stable:$VERSION"
-            # Only push if version tag does not exist
-            if [[ "$PUSH" == "true" ]]; then
-              echo "Will push $FQIN"
-              echo "::set-output name=podman_push::true"
-              echo "::set-output name=podman_fqin::${FQIN}"
+            # quay.io/podman/stable:vX.X.X and :latest
+            if ! fgrep -qx "$VERSION" <<<"$ALLTAGS"; then
+              # Only push if version tag does not exist
+              FQIN="$PODMAN_QUAY_REGISTRY/stable:$VERSION,$PODMAN_QUAY_REGISTRY/stable:latest"
             else
-              echo "Not pushing, $FQIN already exists."
+              FQIN="$PODMAN_QUAY_REGISTRY/stable:latest"
```

Comment:
I've been trying hard to understand this, and unless I'm missing something fundamental, I think this is wrong. IIUC the logic here is:
```
if there's a new tag vX.Y.Z
    push podman/stable:vX.Y.Z
else
    push podman/stable:latest
```
Is my understanding correct? If so, I really think this is wrong. It violates POLA. What I believe should happen is:
```
if there's a new tag vX.Y.Z
    push podman/stable:vX.Y.Z
    push the same image as podman/stable:latest
```
That is: `stable:latest` should always be identical to a tagged image.

Or have I misunderstood something?

Reply:
Thanks for taking a closer look, it's easily possible I've got it wrong.  However, the description in the README.md is correct:

1. If new version (doesn't exist in quay), push it ***and*** the `latest` tag[1]
2. Otherwise, only push `latest` tag.

That said, I'll go double-check this section and fix/add comments accordingly.

[1] Note: It's *possible* a new out-of-order version is pushed to master, in which case "latest" will briefly be a lie (until the next day, when the job runs again).  I'm simply ignoring this corner-case for now.

Reply:
Okay, it's a bigger diff now, but I think it's much clearer this way.  PTAL @edsantiago